docs/sql-guide/statements/statement-database-drop.md

```diff
@@ -0,0 +1,32 @@
+---
+title: DROP DATABASE
+layout: default
+parent: Statements
+grand_parent: SQL guide
+nav_order: 7
+---
+
+# DROP DATABASE statement
+
+Drops the database from featurebase cloud. Note this statement only pertains to FeatureBase Cloud.
```

Comment:
This is a great example of where we can use an includes. Can we do the following:

1. Change this to just " Drops the database from FeatureBase Cloud."
2. Add a line below it " {% include /cloud/cloud-only-note.md%}
3. add a new file in _includes: /cloud/cloud-only-warning.md that contains the following:

> {: .note }
This statement only pertains to FeatureBase Cloud.